standard-integration/server.js

```diff
@@ -25,4 +25,6 @@ app.post("/api/orders/:orderID/capture", async (req, res) => {
   }
 });
 
-app.listen(8888);
+app.listen(8888, () => {
```

Comment:
Thanks for making this change! Some platforms like Heroku leverage `process.env.PORT`. Can we check that first and then default to 8888 like so?
```js
const port = process.env.PORT || 8888;
app.listen(8888, () => {
  console.log(`Server listening at http://localhost:${port}/`);
});

```

Reply:
oo i like that!!